services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -129,4 +129,9 @@ class HankeControllerTest {
         Assertions.assertThat(responseHanke.body?.nimi).isEqualTo("hankkeen nimi")
     }
 
+    // TODO: test that checks that tyomaa- and haitat-fields can be sent to controller and come back in the response
```

Comment:
Are we going to do this? Isn't this the part which is under implementation in the original Jira item? So maybe either implement the test or remove the TODO comment? You can of course push this to dev and continue, if otherwise the set is OK.

Reply:
Yes, still going to do it, as part of the ongoing ticket. The PR was just getting too big.